src/igvfd/tests/test_types_publication.py

```diff
@@ -0,0 +1,8 @@
+import pytest
+
+
+def test_publication_year(testapp, publication):
```

Comment:
can you please add a test to make sure the publication without date_published works ok?